src/_locales/cs.json

```diff
@@ -0,0 +1,114 @@
+{
+  "audio-device-communications": "Vychozí komunikační zařízení",
+  "audio-device-default": "Výchozí systémové zařízení",
+  "audio-device-unknown": "Neznámé zařízení",
+
+  "button-text-ok": "OK",
+  "button-text-cancel-alarm": "Zrušit budík",
+  "button-text-click-here-to-set-alarm": "Klikněte zde pro nastavení budíku",
+  "button-text-close": "Zavřít",
+  "button-text-dont-tell-me-again": "Znovu neupozorňovat",
+  "button-text-lets-do-this": "Pojďme do toho!",
+  "button-text-lets-go": "Tak jdem na to!",
+  "button-text-not-now": "Ne nyní",
+
+  "label-about": "O Aplikaci",
+  "label-alarm": "Budík",
+  "label-desktop-settings": "Nastavení Desktop Přehrávače",
+  "label-donate": "Podpořit",
+  "label-help" : "Nápověda",
+  "label-issues": "Problémy s aplikací",
+  "label-learn-more": "Zjistit více",
+  "label-loading-devices": "Načítání zařízení...",
+  "label-pause-after-song": "Pozastavit po přehrání této skladby",
+  "label-quit": "Ukončit",
+  "label-show-lyrics": "Zobrazit text skladby (Beta)",
+  "label-unknown-artist": "Neznámý umělec",
+  "label-unknown-song": "Neznámá skladba",
+  "label-version": "Verze",
+
+  "lastfm-login-authorized": "Přihlášen",
+  "lastfm-login-authorizing": "Probíhá ověření",
+  "lastfm-login-button-text": "Přihlásit se do služby Last.FM",
+  "lastfm-login-error": "Vypadá to, že jste opustili přihlašovací proces. Pokud se tak nestalo, prosím, opakujte přihlášení",
+  "lastfm-login-not-authorized": "Nepřihlášen",
+  "lastfm-logout-button-text": "Odhlásit se ze služby Last.FM",
+
+  "lyrics-failed-message": "Omlouváme se, ale nenašly se žádné texty pro tuto skladbu",
+  "lyrics-loading-message": "Načítání textu skladby...",
+  "lyrics-no-song-message": "Právě nehraje žádná skladba",
+
+  "message-pausing-after-song": "Pozastavení po této skladbě.",

```

Comment:
We need context for this one because I have honestly not found the function.
What the english text is suggesting is that this is an info message to inform of what will happen after the current song. 
What @ihiroaki is suggesting is that this is a name of a command (perhaps a button).
If I am correct the possible modification would rather be:
a) "Pozastavení po přehrání této skladby"
b) "Bude pozastaveno po přehrání této skladby"
If @ihiroaki is correct then his suggestion is acceptable.

@MarshallOfSound Could you point us as to where this text is displayed and how to test this feature so that we can find the correct wording: